testsuite/grpc-tests/pom.xml

```diff
@@ -123,14 +126,17 @@
         <dependency>
             <groupId>dev.resteasy.grpc</groupId>
             <artifactId>grpc-bridge</artifactId>
+            <version>1.0.0.Alpha6-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>dev.resteasy.grpc</groupId>
             <artifactId>grpc-bridge-runtime</artifactId>
+            <version>1.0.0.Alpha6-SNAPSHOT</version>
```

Comment:
We should use `${project.version}` here.

Reply:
I removed the line ... it inherits the version.